app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class ProjectSerializer < ActiveModel::Serializer
+  attributes :id, :name, :slug, :owner_id, :created_at, :updated_at, :deleted_at, :team_id
```

Comment:
Would be nice to find a way to let the serializer know whether to include them or not. We don’t want to load them all on fetching all projects, just fetching an individual project?